src/store/index.js

```diff
@@ -27,29 +21,23 @@ export default new Vuex.Store({
     setSearchQuery: (state, value) => {
       state.searchQuery = value
     },
-    setFilteredMovies: (state, value) => {
-      state.filteredMovies = value
+    setMovies: (state, value) => {
+      state.movies = value
     }
   },
   actions: {
-    updateFilteredMovies: ({commit, state}) => {
-      const filteredMovies = state.movies.filter(movie => {
-        if (state.searchField === 'genre') {
-          return movie.genres.includes(state.searchQuery)
-        }
-        if (state.searchField === 'title') {
-          return movie.title.includes(state.searchQuery) || movie.overview.includes(state.searchQuery)
-        }
+    loadMovies: async ({commit}) => {
+      let newVar = await movieApi.getMovies();
```

Comment:
Consider using "const" for variables that do not change in the current scope + use meaningful naming.